src/links.ml

```diff
@@ -1,157 +1,130 @@
+open Utils
```

Comment:
Avoid opening modules. It makes code more difficult to read and grep. Prefer fully qualified names or local opens.